ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -447,6 +461,17 @@ function WorkloadCvesOverviewPage() {
                             Manage watched images
                         </Button>
                     )}
+                    {isOnDemandReportsVisible && (
+                        <FlexItem>
+                            <CreateReportDropdown
+                                isOpen={isCreateReportDropdownOpen}
+                                setIsOpen={setIsCreateReportDropdownOpen}
+                                onSelect={() => {
+                                    setIsCreateOnDemandReportModalOpen(true);
```

Comment:
This will evolve to open the right Modal (On-demand reports or Config reports) based on the selected value. Since there's only one now, I open the only one available.